src/agent/src/random.rs

```diff
@@ -10,6 +10,7 @@ use nix::sys::stat::Mode;
 use std::fs;
 use std::os::unix::io::{AsRawFd, FromRawFd};
 use tracing::instrument;
+use crate::util::IoctlRequestType;
```

Comment:
@ruocco We typically expect an explanation for the change in the commit description. Can you add that explaning the need for this change.